src/mesido/workflows/grow_workflow.py

```diff
@@ -110,6 +110,24 @@ def solver_options(self):
         return options
 
 
+class SolverCPLEX:
```

Comment:
Mention this addition in the CGANGELOG as well

Reply:
updated